src/bio2bel/sources/intact.py

```diff
@@ -0,0 +1,645 @@
+    'psi-mi:"MI:0213"(methylation reaction)': ProteinModification('Me'),
+    'psi-mi:"MI:0556"(transglutamination reaction)': ProteinModification('Gln'),
+    'psi-mi:"MI:0220"(ubiquitination reaction)': ProteinModification('Ub'),
+    'psi-mi:"MI:0192"(acetylation reaction)': ProteinModification('Ac'),
+    'psi-mi:"MI:0557"(adp ribosylation reaction)': ProteinModification('ADPRib'),
+    'psi-mi:"MI:0567"(neddylation reaction)': ProteinModification('Nedd'),
+    'psi-mi:"MI:0210"(hydroxylation reaction)': ProteinModification('Hy'),
+    'psi-mi:"MI:0945"(oxidoreductase activity electron transfer reaction)': ProteinModification('Red'),
+
+
+    'psi-mi:"MI:1250"(isomerase reaction)': ProteinModification(
+        name='isomerization',
+        namespace='MOP',
+        identifier='0000789',
+    ),
+
+    'psi-mi:"MI:1237"(proline isomerization reaction)': ProteinModification(
+        name='protein peptidyl-prolyl isomerization',
+        namespace='GO',
+        identifier='0000413',
+    ),
+    'psi-mi:"MI:0193"(amidation reaction)': ProteinModification(
+        name='protein amidation',
+        namespace='GO',
+        identifier='0018032',
+    ),
+    'psi-mi:"MI:1148"(ampylation reaction)': ProteinModification(
+        name='protein adenylylation',
+        namespace='GO',
+        identifier='0018117',
+    ),
+    'psi-mi:"MI:0214"(myristoylation reaction)': ProteinModification(
+        name='protein myristoylation',
+        namespace='GO',
+        identifier='0018377',
+    ),
+
+    'psi-mi:"MI:0211"(lipid addition)': ProteinModification(
+        name='protein lipidation',
+        namespace='GO',
+        identifier='0006497',
+    ),
+    'psi-mi:"MI:1143"(aminoacylation reaction)': ProteinModification(
+        name='tRNA aminoacylation',
+        namespace='GO',
+        identifier='0043039',
+    ),
+
+    'psi-mi:"MI:0883"(gtpase reaction)': ProteinModification(
+        name='GTPase activity',
+        namespace='GO',
+        identifier='0003924',
+    ),
+
+    'psi-mi:"MI:0882"(atpase reaction)': ProteinModification(
+        name='ATPase activity',
+        namespace='GO',
+        identifier='0016887',
+    ),
+
+}
+
+PROTEIN_DECREASES_MOD_DICT: Mapping[str, ProteinModification] = {
+    'psi-mi:"MI:0197"(deacetylation reaction)': ProteinModification('Ac'),
+    'psi-mi:"MI:0204"(deubiquitination reaction)': ProteinModification('Ub'),
+    'psi-mi:"MI:0203"(dephosphorylation reaction)': ProteinModification('Ph'),
+    'psi-mi:"MI:0569"(deneddylation reaction)': ProteinModification('Nedd'),
+    'psi-mi:"MI:0871"(demethylation reaction)': ProteinModification('Me'),
+}
+
+INTACT_OMIT_INTERACTIONS = {
+    'psi-mi:"MI:1110"(predicted interaction)',
+}
+
+EVIDENCE = 'From IntAct'
+
+MODULE_NAME = 'intact'
+VERSION = '2020-03-31'
+URL = f'ftp://ftp.ebi.ac.uk/pub/databases/intact/{VERSION}/psimitab/intact.zip'
+
+UNIPROTKB = 'uniprotkb'
+
+
+def _process_pmid(s: str = '|', prefix: str = 'pubmed:') -> str:
```

Comment:
you're lucky you have a human debugger. The definition and usage of `s` is nonsensical in this function